src/calendly.tsx

```diff
@@ -71,7 +71,13 @@ export type PageSettings = Optional<{
   hideEventTypeDetails: boolean;
   /**
    * @description This setting is only available for Calendly users on the Pro plan. Use this setting to change your Calendly scheduling page's background color.
-   * @example 00a2ff
+   * @example true
+   * @see {@link https://help.calendly.com/hc/en-us/articles/223147027-Embed-options-overview#3} for further information.
+   */
+  hideGdprBanner: boolean;
+  /**
+   * @description The General Data Protection Regulation governs data protection in the EU and EEA. Certain Calendly integrations require access to cookies with user information. If you do not embed the GDPR banner, users in those areas will not have the ability to give their consent in order to access integrations such as Google Analytics, Facebook Pixel, PayPal, and Stripe.
```

Comment:
Can you move this comment above `hideGdprBanner`? It looks like it's currently using the `backgroundColor` description.

<img width="1219" alt="image" src="https://user-images.githubusercontent.com/33756113/97980484-da556580-1d9e-11eb-9e6c-f012b093619a.png">

Reply:
Great catch!